testdata/p4_14_samples_outputs/01-BigMatch-first.p4

```diff
@@ -1,4 +1,5 @@
 #include <core.p4>
+#define V1MODEL_VERSION 20200408
```

Comment:
We could have the test script set this flag.

Reply:
It's the toP4 dumping code that sets/outputs this -- whenever you dump a P4 program that included v1model.p4, it also dumps a #define specifying which version of v1model it was that was included.